client/components/homePage/GenreCaro.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { Image, Dropdown } from 'react-bootstrap';
```

Comment:
I think I saw that genrecaro and trendingcaro got deleted in favor of different carousel components on the home page? Unless something changed.

Reply:
resolved